helm/dagster/README.md

```diff
@@ -24,10 +24,10 @@ helm install my-release . \
 
 ### Installing a specific Chart version
```

Comment:
This isn't you, but since you changed `Chart` > `chart` in the copy that follows, could you also change it in the heading so it's consistent?